src/handlers/no_merges.rs

```diff
@@ -102,11 +104,25 @@ pub(super) async fn handle_input(
         $ # delete any merge commits in the editor that appears
         $ git push --force-with-lease
         ```
+        "
+        .to_string()
+    };
 
+    let since_last_posted = if state.data.mentioned_merge_commits.is_empty() {
+        ""
+    } else {
+        " (since this message was last posted)"
+    };
+    write!(
+        message,
+        "
+    
         The following commits are merge commits{since_last_posted}:
+        "
+    )
+    .unwrap();
 
-    "
-    );
+    let mut should_send = false;
```

Comment:
GitHub won't let me comment on a line outside of the diff, but the `for` loop just below here needs to use `writeln!` instead of `write!`.